e2e/commands/start/start_test.go

```diff
@@ -46,6 +49,50 @@ var _ = Describe("running binary Horusec with start parameter", func() {
 		})
 	})
 
+	When("global flag --log-level is passed", func() {
+		BeforeEach(func() {
+			flags = map[string]string{
+				testutil.StartFlagProjectPath: configFilePath,
+				testutil.GlobalFlagLogLevel: enums.TraceLevel.String(),
+			}
+		})
+
+		It("Checks if the log level was set as trace", func() {
+			Expect(session.Out.Contents()).To(ContainSubstring(`\"log_level\": \"trace\"`))
+		})
+	})
+
+	When("global flag --config-file-path is passed", func() {
+		var configFilePathToTest = filepath.Join(configFilePath, "horusec-config.json")
+
+		BeforeEach(func() {
+			flags = map[string]string{
+				testutil.StartFlagProjectPath: configFilePath,
```

Comment:
@oliveirafelipezup  is working on another feature to fix the name `fileConfigPath` to `projectPath`